tao/tao_bridge/tf/defs.cc

```diff
@@ -20,10 +20,10 @@ namespace tao {
 
 const char* const kXlaMustCompileAttr = "_TaoXlaMustCompile";
 
-const char* const kXlaCompileAttr = "_TaoXlaCompile";
+const char* const kXlaCompileAttr = "_XlaCompile";
 
 // User-provided through jit_scope APIs. Effective only when auto_jit is OFF.
-const char* const kXlaScopeAttr = "_TaoXlaScope";
+const char* const kXlaScopeAttr = "_XlaScope";
```

Comment:
So we always choose to use the same attribute as XLA

Reply:
I added some attr `kReuseXla..` to reuse XlA flags since we can reuse XLA API to make compilation scope, And I  
leave a  TODO comment here, maybe we need some `kDisc...` attributes and relate Python API in the future.